cmd/snap/cmd_wait.go

```diff
@@ -117,21 +116,6 @@ func (x *cmdWait) Execute(args []string) error {
 	snapName := string(x.Positional.Snap)
 	confKey := x.Positional.Key
 
-	// This is fine because not providing a confKey is unsupported so this
-	// won't interfere with supported uses of `snap wait`.
-	if snapName == "godot" && confKey == "" {
-		switch rand.Intn(10) {
```

Comment:
Slightly sad to remove this (and the other)  - couldn't we simply use "randutil.Intn(10)" instead of "rand.Intn(10)"?

Reply:
yes, but they actually confused me a little, it was not immediately clear what this code was about, so to keep them we would have needed to refactor/comment them more,  at that point the cost seemed a bit high, they are kind of fun but not that fun if you are trying to read/understand the code